packages/workbox-google-analytics/src/index.js

```diff
@@ -56,83 +49,122 @@ import replayQueuedRequests from './lib/replay-queued-requests.js';
  * // might be useful is if you wanted to track the amount of time that elapsed
  * // between when the hit was attempted and when it was successfully replayed.
  * workbox.googleAnalytics.initialize({
- *   hitFilter: searchParams =>
+ *   hitFilter: (params) =>
  *     // Sets the `qt` param as a custom metric.
```

Comment:
Stupid comment: but could you add a small depscription of what 'qt' and 'cm1' is?